doc/Getting-started/index.md

```diff
@@ -65,543 +56,204 @@ To use Kaa Sandbox, your system must meet the following minimum system requireme
-    }
+Paste next line in Sandbox terminal:
 
-    - (void)onStarted {
-      NSLog(@"%@ Kaa SDK client started", TAG);
-    }
-    @end
+```sh
+sudo /usr/lib/kaa-sandbox/bin/change_kaa_host.sh <new host/ip>
 ```
-</div></div>
-
-You can find the project source code (including Java, C, C++ and Objective-C) in the attached [archive](attach/kaa-first-app.zip).
-
-- **For Java**, the project is built using Apache Maven. Note that the downloaded SDK must be placed into the lib folder in order for the build to work. Import this project into your IDE as a Maven project and launch the _NotificationSystemTestApp_ application. Once the application is launched, you will see the following output:  
-
-    [...  INFO  NotificationSystemTestApp]  Kaa SDK client started!
-
-- **For C, C++ and Objective-C**, place the downloaded SDK into the _libs_ folder and then run _build.sh_ or _build.bat_ script depending on your platform. Launch the _demo_client_ application. Once the application is launched, you will see the following output:  
-
-    Kaa SDK client started!
-
-## Create notification topic
-
-To send your first Kaa notification, you need to create a notification topic and assign this topic to the default endpoint group.
 
-To create a notification topic, proceed as follows:
+<br>
 
-1. Open the relevant **Notification topics** window (**Applications =\> My First Kaa Application =\> Notification topics**) and click **Add new notification topic**.
-2. In the **Add notification topic** window, fill in the fields as shown in the following screenshot and then click **Add**.  
-![](attach/image2014-7-28 13-38-3.png)
-
-**NOTE:** We set the topic as mandatory in order to automatically subscribe the client application to notifications on this topic.
-
-Once the topic is created, we will assign it to the default endpoint group, which contains all endpoints, including endpoints with our application.
-
-To assign a notification topic to the default endpoint group, proceed as follows:
-
-1. In the relevant **Endpoint groups** window (**Applications=\>My First Kaa Application=\>Endpoint groups**), select the _All_ group.
-2. In the **Endpoint group details** window, click **Add notification topic** at the bottom of the window.
-3. In the **Add topic to endpoint group** window, select the recently created notification topic and then click **Add**.  
-![](attach/image2014-7-28 13-43-38.png)
-
-After the topic is added to the endpoint group, you will see the following output in the application:
-
-    [...  INFO  NotificationSystemTestApp]  Topic list updated!
-    [...  INFO  NotificationSystemTestApp]  Received topic with id X and name Notification Topic
+</div></div>
 
-This is the first update from the Kaa server that provides the client with the information about the changes in the topic list. Now you can send notifications on this topic, and they will be delivered to your application.
+# Your first Kaa application
```

Comment:
This name collides with standalone page  _Your first Kaa application_ 
Consider using another name for this section